packages/applications-service-api/__tests__/integration/applications.test.js

```diff
@@ -319,6 +319,7 @@ describe('/api/v1/applications', () => {
 							{
 								OR: [
 									{ projectName: { contains: 'London Resort' } },
+									{ projectNameWelsh: { contains: 'London Resort' } },
```

Comment:
Would be great to have a test on integration and repository level to only search welsh fields if the feature flag is enabled and a test to ensure we do not search those fields if it is not enabled. (Assuming my understanding of the feature flag is correct).